aws-api-request.js

```diff
@@ -138,7 +137,7 @@ function awsApiRequest(options, retryAttempt = 0) {
                     //2~8 * 100 = 25600
                     //2~9 * 100 = 51200
 
-                    if (retryAttempt > MAX_RETRY_COUNT) {
+                    if (retryAttempt > maxBackoffRetries) {
                         //Give them the error result, the caller can then deal with it...
                         console.warn(`Retry attempt exceeded max retry count (${MAX_RETRY_COUNT})... Giving up...`);
```

Comment:
and use your new variable in the `console.warn`
```suggestion
                        console.warn(`Retry attempt exceeded max retry count (${maxBackoffRetries})... Giving up...`);
```

Reply:
Done in #96 where I have write access :slightly_smiling_face: